app/src/main/java/org/p2p/wallet/home/ui/select/SelectTokenFragment.kt

```diff
@@ -62,4 +67,32 @@ class SelectTokenFragment : BaseFragment(R.layout.fragment_select_token) {
             emptyTextView.isVisible = isEmpty
         }
     }
+
+    override fun onQueryTextSubmit(query: String?): Boolean = false
+
+    override fun onQueryTextChange(newText: String?): Boolean {
+        val searchText = newText.orEmpty()
+        if (searchText.length < QUERY_MIN_LENGTH) {
+            tokenAdapter.setItems(tokens)
```

Comment:
now when we have another type of list of tokens, it's better to name this field as `allTokens`